src/Resources/Loaders.h

```diff
@@ -59,7 +59,8 @@ struct L3DAnimLoader final: BaseLoader<L3DAnim>
 
 struct LevelLoader final: BaseLoader<Level>
 {
-	[[nodiscard]] result_type operator()(FromDiskTag, const std::string& name, const std::filesystem::path& path) const;
+	[[nodiscard]] result_type operator()(FromDiskTag, const std::string& name, const std::filesystem::path& path,
+	                                     bool isCampaign) const;
```

Comment:
For better readability, strong typing and general style, I would always replace a `bool` with an `enum class` in cases like these.